libs/portfolio/domain/src/lib/exception/portfolio-already-archived.exception.ts

```diff
@@ -1,4 +1,6 @@
-export class PortfolioAlreadyArchivedException extends Error {
+import { AggregateException } from '@pl-oss/domain-core';
+
+export class PortfolioAlreadyArchivedException extends AggregateException {
```

Comment:
Please rename the file and directory as `aggregate-exception`.
```suggestion
export class PortfolioAlreadyArchivedAggregateException extends AggregateException {
```

Reply:
Resolved in https://github.com/venusvavadiya/ticksbook-main/pull/18/commits/edcba7bf84a94dfbfc5c31773c026132ceebe79c